tests/tests.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+current_directory=$(pwd)
+test_framework_directory="$current_directory/jenkinsfile-runner-test-framework"
+sh_unit_directory="$test_framework_directory/shunit2"
+
+version="256.0-test"
+CWP_version="1.3"
+jenkinsfile_runner_tag="jenkins-experimental/jenkinsfile-runner-test-image"
+
+downloaded_cwp_jar="to_update"
+
+. $test_framework_directory/utilities/utils.inc
+. $test_framework_directory/utilities/cwp/custom-war-packager.inc
+. $test_framework_directory/utilities/jfr/jenkinsfile-runner.inc
+
+oneTimeSetUp() {
+  downloaded_cwp_jar=$(download_cwp "$test_framework_directory" "$CWP_version")
+}
+
+setUp() {
+  echo "Initializing the test case. Ignoring so far"
+}
+
+test_with_tag() {
```

Comment:
AFAICT this method body should be a method of the test framework in https://github.com/jenkinsci/jenkinsfile-runner-test-framework/pull/1 . Looks to be a standard smoke test method

Reply:
You mean it's not a real test?

Reply:
IIUC, @oleg-nenashev you mean those tests are good enough to be smoke tests for the framework. My initial intention was to have the structure and one test that could be used as exemple. I'll put these tests in the framework and just add a totally fake test here that could be removed once the real tests are improved.